src/list.js

```diff
@@ -1,31 +1,72 @@
 import React, {Component} from 'react'
+import Grid from '@material-ui/core/Grid'
 import List from '@material-ui/core/List'
 import ListItem from '@material-ui/core/ListItem'
 import ListItemText from '@material-ui/core/ListItemText'
-import { withStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import { withStyles, createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles'
+import green from '@material-ui/core/colors/green'
 
-const styles = {
+const styles = theme => ({
   list: {
-    maxWidth: 500,
-    maxHeight: 800,
-    overflow: 'auto',
-    marginLeft: 350,
+    width: 560,
     border: '1px solid silver',
     borderRadius: '0.25rem'
+  },
+  button: {
+    color: theme.palette
   }
-}
+})
+
+const theme = createMuiTheme({
+  palette: {
+    primary: green,
+  },
+  typography: {
+    useNextVariants: true,
+  }
+})
 
 class IngredientList extends Component {
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    this.props.getRecipes()
+  }
+
   render() {
     const { classes, ingredientList } = this.props
     return (
-    <List className={classes.list}>
-      {ingredientList.map(item => {
-        return <ListItem key={item.id}>
-          <ListItemText primary={item.ingredient}/>
-        </ListItem>
-      })}
-    </List>
+    <Grid container
+      alignItems="center"
+      direction="column"
+      spacing={40}
+      >
+      <Grid item>
+          <List className={classes.list}>
+          {ingredientList.map(item => {
+            return <ListItem key={item.id}>
+              <ListItemText primary={item.ingredient}/>
+            </ListItem>
+          })}
+          </List>
+      </Grid>
+      {ingredientList.length > 0 &&
+      <Grid item>
+        <MuiThemeProvider theme={theme}>
```

Comment:
Use this `MuiThemeProvider` in the render method of your `App` instead of just inside this list. That way the theme styles are applied globally.